app/src/main/java/com/nextcloud/talk/chat/BubbleActivity.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Nextcloud Talk - Android Client
+ *
+ * SPDX-FileCopyrightText: 2025 Alexandre Wery <nextcloud-talk-android@alwy.be>
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+package com.nextcloud.talk.chat
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import com.nextcloud.talk.R
+import com.nextcloud.talk.utils.bundle.BundleKeys
+
+class BubbleActivity : ChatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        supportActionBar?.setDisplayHomeAsUpEnabled(false)
+        supportActionBar?.setDisplayShowHomeEnabled(false)
+    }
+
+    override fun onPrepareOptionsMenu(menu: android.view.Menu): Boolean {
+        super.onPrepareOptionsMenu(menu)
+
+        menu.findItem(R.id.create_conversation_bubble)?.isVisible = false
+        menu.findItem(R.id.open_conversation_in_app)?.isVisible = true
+
+        return true
+    }
+
+    override fun onOptionsItemSelected(item: android.view.MenuItem): Boolean =
+        when (item.itemId) {
+            R.id.open_conversation_in_app -> {
+                openInMainApp()
+                true
+            }
+            else -> super.onOptionsItemSelected(item)
+        }
+
+    private fun openInMainApp() {
+        val intent = Intent(this, ChatActivity::class.java).apply {
+            putExtras(this@BubbleActivity.intent)
+            flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TOP
```

Comment:
multiple instances of the app are opened when "open is app" is used.
https://developer.android.com/develop/ui/views/notifications/bubbles#launching-activities